File: test1.py
import time

start_time = time.time()
iteration = 1_000_000_000
dop = 1
total = 0
i = 0
while i < iteration:
    total += 1 / dop
    dop += 2
    total -= 1 / dop
    dop += 2
    i += 1

pi = total * 4
end_time = time.time()

print("Количество Итераций: ", iteration)
print("Число Пи: ", pi)
print("Время работы:", end_time - start_time)
# By ArisNow0

File: test1.c
#include <stdio.h>
#include <time.h>

int main() { 
    clock_t start = clock(); 

    long long iteration = 100000000000LL;
    double dop = 1.0;
    long long i = 0;
    double total = 0.0;

    while (i < iteration) {
        total += 1.0 / dop;
        dop += 2.0;
        total -= 1.0 / dop;
        dop += 2.0;
        ++i;
    }

    double pi = total * 4.0;

    clock_t end = clock();
    double diff = (double)(end - start) / CLOCKS_PER_SEC;

    printf("Количество Итераций: %lld\n", iteration);
    printf("Число Пи: %.15f\n", pi);
    printf("Время: %f секунд\n", diff);

    return 0;
} // By ArisNow0